src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -807,7 +807,20 @@ public enum LineWrapMode
         /// character will move character by character to the next line.
         /// </summary>
         /// <since_tizen> 4 </since_tizen>
-        Character
+        Character,
+
+        /// <summary>
+        /// Hyphenation mode will move part of the word (at possible hyphen locations)
+        /// to the next line and draw a hyphen at the end of the line.
+        /// </summary>
+        /// <since_tizen> ? </since_tizen>
+        Hyphenation,
```

Comment:
If you want to add Public API, you should do ACR process. If not, add it as a hidden API. ,
you can add tag '[EditorBrowsable(EditorBrowsableState.Never)]' to make hidden API.

Reply:
I have added the the hidden API tags